R/read_gt3x.R

```diff
@@ -80,6 +81,34 @@ read_gt3x <- function(
     }
 
   }
+  if ("RAW" %in% names(log)) {
+    if ("Timestamp" %in% names(log$RAW)) {
+      check = anyDuplicated(log$RAW$Timestamp)
+      if (any(check)) {
+        warning(
+          "Duplicated timestamps in the data, this usually indicates an error",
+          call. = FALSE
+        )
+      }
+    }
```

Comment:
I think this can be cleaned up. It would also be nice to give users the option of bypassing the checks with a new argument (maybe `extra_checks` as I've labeled it below), similar to what we've done for `flag_idle_sleep` and `cleanup`. Default value could probably be `extra_checks = TRUE`.

```suggestion
if (!is.null(log$RAW$Timestamp) & extra_checks) {

  if (verbose) cat("\n  Running some extra checks")
  
  if (anyDuplicated(log$RAW$Timestamp)) warning(
    "Duplicated timestamps in the data. This usually indicates an error",
    call. = FALSE
  )
```

Reply:
Fixed